README.md

```diff
@@ -53,41 +80,50 @@ When you ask your assistant to “Turn on the light”, it will use the auth bea
     "automation" :
     {
      "name": "automation",
-     "url": "https://YOUR-URL/openhabGoogleAssistant"
+     "url": "https://us-central1-YOUR-APPLICATION.cloudfunctions.net/openhabGoogleAssistant"
```

Comment:
Please keep it generic, since it might change at any time. Google Cloud Functions is in beta. So lets make it generic when it comes to URLs.

Reply:
I'm going to change this to `YOUR-OPENHAB-CLOUD-URL` as I've seen people be confused about what servers go where.